temporalio/sdk-typescript
# fix: Get rid of istanbul-instrumenter-loader

## What changed

In @temporalio/nyc-test-coverage:
- Remove dependency on istanbul-instrumenter-loader
- Replaced by a private webpack module loader
- Add integration tests
- Modify rules to make sure that all user source files are included in coveerage

## Why

- 'istanbul-instrumenter-loader' was causing inclusion of vulnerable package ' loader-utils@1.4.0'
- project 'istanbul-instrumenter-loader' is no longer maintained (it has actually been archived)
- fix #994, fix #911, fix #872
